features/step_definitions/work_related_steps.rb

```diff
@@ -159,25 +159,40 @@
   step %{I should see "Followup by remixer" within ".afterword .children"}
 end
 
+Then /^I should see the related work listed on the original work$/ do
```

Comment:
I think this and the other new/changed steps in the file should be using [cucumber expressions](https://github.com/cucumber/cucumber-expressions#readme) instead of regex.